coins.json

```diff
@@ -9,8 +9,13 @@
         "curve": "secp256k1",
         "publicKeyType": "secp256k1Extended",
         "explorer": "https://explorer.nebulas.io/#/tx/",
-        "url": "https://nebulas.io",
-        "rpcNodeInfo": "https://github.com/nebulasio/go-nebulas"
+        "info": {
+            "url": "https://nebulas.io",
+            "node": "https://github.com/nebulasio/go-nebulas",
+            "publicNode": "https://mainnet.nebulas.io",
+            "docs": "https://wiki.nebulas.io",
+            "rpcDocs": "https://wiki.nebulas.io/en/latest/dapp-development/rpc/rpc.html"
+        }
```

Comment:
It would be nice to also add the url of our node. For example:

`trustNode`: `https://nebulas-rpc.trustwalletapp.com`